src/modules/addPopup.js

```diff
@@ -1,118 +1,130 @@
-// const btn = document.querySelector('.btn_comment');
-
-class commentsPage {
-    URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tC8kuxLLK9p84n1K7Qba/comments';
-
-    constructor(shows, btn) {
-      [this.show] = shows.moviesarray.filter((show) => show.id === +btn.id);
-      this.GetShow = new GetShow(this.URL);
-      this.commentsArray = [];
-    }
-    // To create template to render the items in the modal
-
-    renderMovies() {
-      const html = `
-  <section class="sectionModal">
-  <p class="close">&times</p>
-  <article class="movies">
-  <img class="moviesImg" src="${this.show.image.medium}" />
-  <div class="movieData">
-      <h3 class="moviesName">${this.show.name}</h3>
-      <div class="item1"> </div>
-       <p class="moviesRow"><span></span>${this.show.summary}</p>
-  </div>
-  </article>
-  
-  <div class=commentContainer>
-  <p> Comments (<span class="commentCounter">p</span>)</p>
-  <div class=commentBox> </div>
-  </div>
-  
-  <div class="writeComments">
-  <p class="writeComment">Write Comment</p>
-  <input type="text"  placeholder="Username" class="inputName"></input>
-  <input type="textarea" placeholder="Comment" class="inputComment"></input>
-  <button type="button" class="btn btn_primary btn_movies">Comment</button>
-  </div> 
-  </section>`;
-
-      const commentsPage = document.querySelector('.commentSection');
-      const popUp = commentsPage.querySelector('.sectionModal');
-      popUp.innerHTML = html;
-
-      this.getComments();
+import getComments from './getComments.js';
+import cancel from '../cancel.png';
+import getShowById from './getShowById.js';
+import postComment from './postComment.js';
+import cleanElement from './cleanElement.js';
+
+// To create template to render the items in the modal
+const renderModal = async (id) => {
+  // Get show
+  const show = await getShowById(id);
+
+  // Create modal content
+
+  // Create elements
+  const divShowModal = document.querySelector('.modal');
+
+  const divImgModal = document.createElement('div');
+  const imgClose = document.createElement('img');
+  const imgShowModal = document.createElement('img');
+
+  const divShowInfo = document.createElement('div');
+  const h3ShowTitle = document.createElement('h3');
+  const pSummary = document.createElement('p');
+
+  const divComments = document.createElement('div');
+  const numComments = document.createElement('span');
+  const ulComments = document.createElement('ul');
+
+  const divAddComment = document.createElement('div');
+  const inpName = document.createElement('input');
+  const inpComment = document.createElement('input');
+  const btnComment = document.createElement('button');
+
+  // Add content and atributes
+  divShowModal.classList.add('modal');
+
+  imgShowModal.src = show.image.medium;
+  imgClose.classList.add('closeModal');
+
+  // Show image
+  divImgModal.classList.add('imageModal');
+  imgClose.src = cancel;
+  divImgModal.appendChild(imgShowModal);
```

Comment:
- [ ] Great job working on the pop up but as I can see the information is a summary and i think you should put four things displaying some information for each movie ✅


- To understand me better I am talking about this part 
![image](https://user-images.githubusercontent.com/58553711/183229577-e8854c7a-a7ad-4988-a1ba-62f67c301be3.png)


- Microverse design
![image](https://user-images.githubusercontent.com/58553711/183229706-e761695a-05b5-45b8-a758-c5f59ef73a26.png)

Reply:
Hello @Reem-lab, thanks for your feedback. I have one doubt we have to use the new pull request against the main to ask for a new code review or do we have to use this one?
Thanks.

Reply:
Hello @VaneCode Yes you have to merge this branch to dev then you can make a new pull request from dev to main then send the pull request to ask for a review 🙏🏻🙏🏻🙏🏻✅✅